README.md

```diff
@@ -1,2 +1,33 @@
 # OpenRefine-metadata-extension
-Extension for OpenRefine to support FAIR metadata
+
+[![Build Status](https://travis-ci.com/FAIRDataTeam/OpenRefine-metadata-extension.svg?branch=master)](https://travis-ci.com/FAIRDataTeam/OpenRefine-metadata-extension)
```

Comment:
Build status on appropriate branch? Now it's displaying the status of `master`?

Reply:
I don't think it is possible to let it recognize on which branch is the README file opened. We can leave out the branch but then it will be latest build. Like this it provides build status of latest push into master that we should maintain "stable". For other branches and PRs we can use GitHub statuses thanks to integration...